cost/scheduled_reports_with_estimates/CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Change log
```

Comment:
Looks like this was copied from the Schedule Report Policy.  It should have only v1.0

Reply:
done

Reply:
I don't see your changes.  You need to remove all versions except v1.0